dashboard/app/assets/javascripts/editor/blockly_editor.js.erb

```diff
@@ -6,10 +6,13 @@
 //= require codemirror/addons/fold/xml-fold
 //= require codemirror/modes/xml
 //= require codemirror/modes/javascript
+//= require codemirror.inline-attach
 //= require_self
 //= require jquery.handsontable.full
-//= require codemirror.inline-attach
-function codeMirror(name, mode, callback) {
+
+$('head').append($('<script src="/blockly/js/marked/marked.js" type="text/javascript"/>'));
```

Comment:
This looks like a very basic use-case of including a dependency that's readily available in just about every single package-manager (e.g., [marked-rails](https://github.com/rosscooperman/marked-rails), [marked](https://www.npmjs.com/package/marked) on npm, etc), so this line is a pattern we probably don't want to keep around anywhere in our codebase for very long. lf we have the time, can you work with Brent to figure out how to include a dependency like this in whatever recommended way is more in line with the work he's been doing on frontend refactoring?